Scripts/Runtime/Data/Collections/UnsafeHeap.cs

```diff
@@ -43,6 +43,20 @@ public struct UnsafeHeap<T, TComparer> : IDisposable
         where T : unmanaged
         where TComparer : unmanaged, IComparer<T>
     {
+        /// <summary>
+        /// Whether this collection has been created or not (had memory been allocated)
+        /// </summary>
+        public bool IsCreated
```

Comment:
Yeah because this class is a copy/paste job of the `NativeHeap` that was written by someone else, we didn't bother to convert over to our style. 

We could, but we aren't going to use this in the future due to #151